linux_os/guide/system/network/network-nftables/set_nftables_loopback_traffic/ansible/shared.yml

```diff
@@ -0,0 +1,38 @@
+# platform = multi_platform_sle
+
+- name: Implement loopback rules
```

Comment:
In alignment to Style Guide, task names must be in [Title case](https://en.wikipedia.org/wiki/Title_case)